packages/pi-heif/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: pi-heif
+  version: 0.21.0
+  top-level:
+    - pi_heif
+source:
+  url: https://files.pythonhosted.org/packages/f5/12/e87b1a7e5b353f885b646ee9c966be74b7db0ae9d68abc712411487353d7/pi_heif-0.21.0.tar.gz
+  sha256: 4902cdb84e75505e1d9abdd5aff1e6dcfebe569ec825162d68a4a399a43689a4
+requirements:
+  run:
+    - cffi
+    - Pillow
+    - libheif
+  host:
+    - cffi
+    - libheif
+build:
+  script: |
+    export LIBHEIF_ROOT=${WASM_LIBRARY_DIR}
+test:
+  imports:
+    - pi_heif
+about:
```

Comment:
Could you add yourself as a recipe maintainer? See [example](https://github.com/pyodide/pyodide/blob/e54b220cf46aaa60b1ba57a6e620191824540df0/packages/libcst/meta.yaml#L16-L18) here.

```yaml
extra:
  recipe-maintainers:
    - airen1986
```